edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/CertificateRenewal.cs

```diff
@@ -97,5 +109,18 @@ void Callback(object _state)
                 this.timer.Change(Timeout.InfiniteTimeSpan, Timeout.InfiniteTimeSpan);
             }
         }
+
+        void PeriodicCallback(object _state)
+        {
+            var currentTime = DateTime.UtcNow;
+            TimeSpan timeToExpire = this.certificates.ServerCertificate.NotAfter - DateTime.UtcNow;
```

Comment:
nit: Couldn't we use the `currentTime` calculated above?

Reply:
ah yes, thanks.